themes/default/data/team/team/joshua-studt.toml

```diff
@@ -2,6 +2,7 @@ id = "joshua-studt"
 name = "Joshua Studt"
 title = "Solutions Architect, Financial Independence Group"
```

Comment:
we should break this into title and company like the rest of them

Reply:
As this has been merged, I opened a new PR to fix 

https://github.com/pulumi/pulumi-hugo/pull/1641